ui/src/tool-panel/track-number/track-number-infobox.tsx

```diff
@@ -225,17 +225,19 @@ const TrackNumberInfobox: React.FC<TrackNumberInfoboxProps> = ({
                             }
                         />
                         <InfoboxField
-                            label={`${t('tool-panel.reference-line.start-coordinates')} ${coordinateSystem.name
-                                }`}
+                            label={`${t('tool-panel.reference-line.start-coordinates')} ${
+                                coordinateSystem.name
+                            }`}
                             value={
                                 startAndEndPoints?.start
                                     ? formatToTM35FINString(startAndEndPoints.start.point)
                                     : ''
                             }
                         />
                         <InfoboxField
```

Comment:
Ei sinällään tämän reviewin asiaa, mutta tätäkin tsx:ää voisi rempata niin että tehdään noita ala-osioita omilla funkkareilla joita vain kutsutaan täältä. Tuolla tapaa saa abstraktiotason pysymään ymmärrettävämpänä ja näitä karkaavia sisennyksiä kuriin. 

Esim tässä voisi ihan hyvin tehdä funkkarin createCoordinateField(point, coordinateSystem) joka palauttaa tuon <InfoboxField ...> ja saisi nämä 2 molemmat tehtyä sillä.

Tuossa ylempänä esim tuon alignmentin tietojen fragmentin luonnin voisi eriyttää myös omaksi funkkariksi.